app/routes/individual_response.py

```diff
@@ -63,10 +65,10 @@ def request_individual_response(schema, questionnaire_store):
         list_item_id=list_item_id,
     )
 
-    if request.method == "GET":
-        return individual_response_handler.handle_get()
+    if request.method == "POST":
```

Comment:
The default fallback option should really always be the call to the get. The only other place I can see this is 3 cases in questionnaire.py, where the logic on the get condition is confusing because it involves a call to the post form. They should probably be changed too.